README.md

```diff
@@ -1 +1,32 @@
-# rent-listings-notifier
\ No newline at end of file
+Rent Listings Notifier
+=====================
+Rent Listings Notifier is a robot set up to be executed every day that goes to Craigslist Website, get a list of rental properties according to the filter pre-configured, saves the list in a DynamoDB, compares the current data from Craigslit with the data saved in the day before and then sends an email with the new rental properties added on Craigslist, the properties added in previous days and the removed ones. The filters that can be passed to the Lambda function are the same avaiable on Craigslist Website, they are: Km from Postal Code, Min. and Max of Price, Min. and Max Bathrooms, Min. and Max FT, has image, bundle duplicates and others. It was built in AWS Lambda NodeJS, DynamoDB, SNS (Simple Notification Service from AWS) and Cloudwatch event used to schedule the execution of the Lambda function.
```

Comment:
"set up to be executed every day" => set up to be executed daily

Instead of using `properties` (you used some throughout  the readme), use `listings` .

`has image` and `bundle duplicates` cannot be passed to the lambda. They are hardcoded.

Reply:
Done.